iron-autogrow-textarea.html

```diff
@@ -47,7 +47,6 @@
       position: relative;
       width: 400px;
       border: 1px solid;
-      padding: 2px;
```

Comment:
Removing this padding renders the text and the cursor outside the border.

Before (in https://elements.polymer-project.org/elements/iron-autogrow-textarea?view=demo:demo/index.html&active=iron-autogrow-textarea):
<img width="238" alt="screen shot 2015-12-14 at 2 58 24 pm" src="https://cloud.githubusercontent.com/assets/1369170/11796803/3b6f0fc6-a273-11e5-834b-e923d0d99d45.png">

Locally, with your code:
<img width="233" alt="screen shot 2015-12-14 at 2 58 31 pm" src="https://cloud.githubusercontent.com/assets/1369170/11796805/3ea9226c-a273-11e5-90f6-6cb91adf1baa.png">